javasphinx/compiler.py

```diff
@@ -40,10 +39,11 @@ def __default_filter(self, node):
         docblocks.
 
         """
+
         if not isinstance(node, javalang.tree.Declaration):
             return False
 
-        if 'private' in node.modifiers:
+        if node.modifiers is not None and 'private' in node.modifiers:
```

Comment:
In fact this verification is not required. It was used because I was calling the compile method for javalang.tree.PackageDeclaration elements, but it is not called anymore.